src/tables.jl

```diff
@@ -0,0 +1,87 @@
+# Tables and TableTraits interface
+
+## Chains
+
+Tables.istable(::Type{<:Chains}) = true
+
+Tables.columnaccess(::Type{<:Chains}) = true
+
+Tables.columns(chn::Chains) = chn
+
+Tables.columnnames(chn::Chains) = (:Iteration, :Chain, Symbol.(names(chn))...)
+
+function Tables.getcolumn(chn::Chains, i::Int)
+    return Tables.getcolumn(chn, Tables.columnnames(chn)[i])
+end
+function Tables.getcolumn(chn::Chains, nm::Symbol)
+    if nm == :Iteration
+        niter, _, nchains = size(chn)
+        return repeat(Base.OneTo(niter), nchains)
+    elseif nm == :Chain
+        chainids = chains(chn)
+        niter = size(chn, 1)
+        return vcat(fill.(chainids, niter)...)
```

Comment:
This should be written as
```suggestion
        return repeat(chainids; inner = niter)
```